rackspace.yaml

```diff
@@ -4,6 +4,7 @@ application-name: Drupal
 application-version: 7.32
 flavor: Multiple Linux servers
 flavor-weight: 1
+popularity: 5
```

Comment:
question on this: how is this template in the top 5 when it's not even returned in the template catalog? drupal-single is the only drupal template in the catalog